src/lib/elements/forms/button.svelte

```diff
@@ -40,11 +37,11 @@
 {#if href}
     <a
         on:click={track}
-        {...attributes}
         {href}
         target={external ? '_blank' : ''}
         rel={external ? 'noopener noreferrer' : ''}
         class="button"
+        class:is-disabled={disabled}
```

Comment:
```suggestion
        class:is-disabled={internalDisabled}
```

Reply:
@TorstenDittmann Not sure this change is needed... if for example, we have links inside of modals they would be disabled needlessly disabled :thinking: